docs/framework/windows-workflow-foundation/whats-new.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "What's New in Windows Workflow Foundation"
+description: Learn about Windows Workflow Foundation in .NET Framework 4 changes to several development paradigms from previous versions.
```

Comment:
```suggestion
description: Learn about Windows Workflow Foundation changes in .NET Framework 4.
```

Reply:
@gewarren - and this: 

Learn about Windows Workflow Foundation changes in .NET Framework 4. Workflows are easier to create, execute, and maintain, and implement new functionality.

Reply:
```suggestion
description: Learn about Windows Workflow Foundation changes in .NET Framework 4. Workflows are easier to create, execute, and maintain.
```